lib/utils.js

```diff
@@ -7,7 +8,7 @@
  * @returns {string} The qualified activity name
  */
 function qualifyActivityName (activityName, packageName) {
-  if (!activityName || !packageName) {
+  if ([activityName, packageName].some((name) => !name || _.includes(name, '*'))) {
```

Comment:
While it's clear what this is doing in the context of this PR, I fear it will not make any sense once just a bit of code. A comment would go a long way.

Reply:
added a comment